### ISSUE TITLE
Page content disappear sometimes

### ISSUE THREAD

**Issue by hebertialmeida** [FolioReader/FolioReaderKit#152]
@tschob I am preparing to swift 3 migration, I think this problem maybe caused on page, center and container changes, could you help me track that?

I have merged you changes and I am not sure it everything is ok on branch `swift23`, seems that it is ok on master.

**Steps to reproduce:**
Click on internal links, sometimes it break on scrolling to next page.
Watch the video https://dl.dropboxusercontent.com/u/944561/Bug-content-FolioReader.mov

P.S. You have to use Xcode 8 to run this branch.

**Comment by tschob** [comment]
@hebertialmeida I took a look in it but didn't solved it for now. It looks strange: If the webview isn't visible it's actually because the `UICollectionViewCell` seems to be gone. If the View Hierachy is captured by the view debugger the collection view cell is there:

**Screenshot 1: Working and visible page**
![bildschirmfoto 2016-09-21 um 09 58 54](https://cloud.githubusercontent.com/assets/3233717/18702733/48540568-7fe3-11e6-97f0-281d1acda873.png)

**Screenshot 2: Page is not visible**
![bildschirmfoto 2016-09-21 um 10 01 15](https://cloud.githubusercontent.com/assets/3233717/18702756/59aebbbe-7fe3-11e6-868c-5a4a2cf4a058.png)

**Comment by tschob** [comment]
Second observation: I can't reproduce it on iOS 9.2. Can you?

Unfortunately I don't have time for further investigation as our project is in a hurry and I need to continue working on that.

**Comment by tschob** [comment]
It's seems that the new prefetching is the problem. I couldn't reproduce the bug by disabling the prefetching.
```
if #available(iOS 10.0, *) {
	collectionView.prefetchingEnabled = false
}
```
This could be the Bugfix. But it may make sense to try to find out what the exact problem is. If it's not an iOS 10 Bug it might be possible to keep the prefetching enabled without a broken folio reader.

**Comment by hebertialmeida** [comment]
That makes sense, I will investigate...

**Comment by hebertialmeida** [comment]
It is not only in swift 2.3, also in 2.2 iOS 10.

**Comment by hebertialmeida** [comment]
@tschob Definitely it is not related to iOS 10, I can reproduce it on iOS 9.3 with Swift 2.2. It is happening on horizontal scroll direction for me.

![screen shot 2016-09-22 at 23 57 08](https://cloud.githubusercontent.com/assets/103670/18772840/8ce079d0-8139-11e6-8237-6a9331d835d1.png)

**Comment by hebertialmeida** [comment]
For now it is fixed using @tschob's solution
```
if #available(iOS 10.0, *) {
    collectionView.prefetchingEnabled = false
}
```
